### ISSUE TITLE
[Feature Request] show Time next to calendar

### ISSUE THREAD

**Issue by lublak** [Vuepic/vue-datepicker#142]
**Is your feature request related to a problem? Please describe.**

Currently you have to switch between calendar and timepicker.

**Describe the solution you'd like**

It would be handy if the timepicker could be displayed right next to the calendar. (as an option)
![image](https://user-images.githubusercontent.com/44057030/190084443-e5dc0db0-9f06-46c0-ae4d-c62ae5d03e53.png)

**Describe alternatives you've considered**

So, basically, there is no alternative but to leave it as it is.

**Comment by Jasenkoo** [comment]
This will not be added, it will just add unnecessary complexity to the picker and it is not intended by design. Also, having the time on the right is probably not the best way to handle input on the mobile device. However, there will be a new slot added `right-sidebar` with new methods and you can implement it on your own if wanted. Same thing for #143 and #144
